packages/react-devtools-shared/src/devtools/views/DevTools.js

```diff
@@ -135,11 +124,9 @@ export default function DevTools({
   defaultTab = 'components',
   enabledInspectedElementContextMenu = false,
   fetchFileWithCaching,
-  loadHookNames,
+  hookNamesModuleLoaderFunction,
```

Comment:
This exists because:
1) Dynamic `import()` have to be static (no `import(variable)`)
2) Webpack 4 awkwardness if the `react-devtools-inline` package itself defines the `import()`.